nexus-repository-p2/src/main/java/org/sonatype/nexus/repository/p2/internal/metadata/ArtifactsXmlAbsoluteUrlRemover.java

```diff
@@ -68,66 +70,149 @@
 public class ArtifactsXmlAbsoluteUrlRemover
```

Comment:
Please change 'since' version to the current used

Reply:
Done